README.md

```diff
@@ -20,10 +20,17 @@
 ember install ember-frost-sidebar
 ```
 ## API
-Coming soon
+Not applicable at the moment.
 
 ## Examples
-Coming soon
+```
+{{#frost-sidebar class="demo-instance"}}
+  <div class="demo-pod-content">
+    <h4>Test content</h4>
+    {{frost-checkbox  checked=true label="Check me"}}
+  </div>
+{{/frost-sidebar}}
+```
 
 ## Development
```

Comment:
Most of the repos have it.  frost-core is a bit different as there is a main page and then a link to each individual component.  Take a look at this one: https://github.com/ciena-frost/ember-frost-modal-input